roles/install/tasks/tower_install.yml

```diff
@@ -15,7 +15,6 @@
 - block:
     # Run the Setup
     - name: "[Tower] Run the Ansible Tower Setup Program"
-      become: true
```

Comment:
@sean-m-sullivan just want to check with you this is a correct change to make. I can't think of a reason this should actually be set to true here. As pointed out by Chris, this shouldn't need to be true, and in some cases will not work if you don't have root access on the local machine (when using a bastion for example). In the situation you actually do want to run this as root locally the you'd run the whole thing as become I guess.

Reply:
that I'm not sure, I'm going to do some testing on this branch just to verify. I know I came accross the root problem this week when standing up a new one.